setup.cfg

```diff
@@ -12,6 +12,7 @@ exclude=
   .git,__pycache__,
   .eggs,*.egg,
   build,
+  ddtrace/__init__.py,
```

Comment:
Why is this excluded from flake8 linting? I can't see how it's related to the rest of this change.

Reply:
can we handle in a separate PR so we have a single change/commit/PR description with motivating factors?

your reasoning seems logical, but feels unrelated to this change overall.

Reply:
In #5428